site2/docs/client-libraries-go.md

```diff
@@ -424,6 +401,7 @@ if err != nil {
 defer consumer.Close()
 
 for i := 0; i < 10; i++ {
+    // may block here
 	msg, err := consumer.Receive(context.Background())
 	if err != nil {
 		log.Fatal(err)
```

Comment:
@labuladong generally Golang use tab for indent. In this patch we mix tabs and spaces, you may keep them consistent.

Reply:
Not really. [These lines](https://github.com/apache/pulsar/blob/a053323e23998d9b7e746933cbce43590ad956b9/site2/docs/client-libraries-go.md?plain=1#L71-L76) are still using tabs. I suggest you replace all leading space in Golang code blocks with tabs.

Reply:
Code in markdown block will treat `\t` as 4 spaces. But you are right, I can use regex to replace them.

Reply:
Yes. I'm OK to use spaces here since Go formatter should be able to format it. My suggestion focuses on consistently - that is, we don't use both.